spec/src/consensus.rs

```diff
@@ -462,17 +462,17 @@ impl Consensus {
                 let denominator = U256::from(next_epoch_length);
                 (diff_numerator / denominator).into_u256()
             } else {
-                let orphan_rate_estimation = RationalU256::one()
-                    / (((&last_orphan_rate + U256::one())
-                        * &epoch_duration_target_u256
-                        * &last_epoch_length_u256
-                        / (&last_orphan_rate * &last_epoch_duration * &next_epoch_length_u256))
-                        .saturating_sub_u256(U256::one()));
-
-                let denominator = if orphan_rate_estimation.is_zero() {
+                let orphan_rate_estimation_recip = ((&last_orphan_rate + U256::one())
```

Comment:
Need a test case to cover it. But the test doesn't need to backport to 0.18, just add to the PR send to develop.